Duplicati.Library.RestAPI/Database/ServerSettings.cs

```diff
@@ -563,17 +565,28 @@ public string ServerListenInterface
             }
         }
 
-        public X509Certificate2 ServerSSLCertificate
+        public bool ServerUseHTTPS
```

Comment:
I think it makes sense to have a general property that explains if HTTPS should be enabled.
I think this should replace the different checks for empty strings.

Reply:
Checks for empty strings are there for backwards compatibility.